docs/source/topic_guides/ffapply.rst

```diff
@@ -0,0 +1,27 @@
+Applying a force field
+======================
+
+The main method you will use in **foyer** is the
+``Forcefield.apply()`` method. There are a few important arguments
+you should understand.
+
+The first several are the ``assert_bond_params``
+``assert_angle_params``, ``assert_dihedral_params``, and
+``assert_improper_params``. These arguments require that the
+supplied force field has parameters for every bond, angle, dihedral,
+and improper in the system. In most cases, if you get an error,
+it means that your force field is missing parameters for one of the
+bonds/angles/dihedrals/impropers in the system. This could be
+because the parameters are missing or because the atom-typing
+(i.e., the SMARTS strings) are incorrect. These arguments are ``True`` by default,
+with the exception of ``insert_improper_params``. In all cases, it
```

Comment:
`insert_improper_params` is the name of the function argument, so it is correct in this case. I think we can leave these as-is, since parameters is a bit verbose.

Reply:
Nice catch @CalCraven . @justinGilmer what are you talking about?

Reply:
my bad, I completely missed the assert vs insert part of the typo. Ignore me :)